keras/preprocessing/image.py

```diff
@@ -34,12 +34,14 @@ def random_shift(x, wrg, hrg, fill_mode="nearest", cval=0.):
     if wrg:
         crop = random.uniform(0., wrg)
         split = random.uniform(0, 1)
-        crop_left_pixels = int(split*crop*x.shape[1])
+        randSign = 2*np.random.randint(2)-1 #allows cropping from left and right both
```

Comment:
PEP8: spaces after operators (e.g. `*`), double space before comment and space after `#`.